src/styles/GlobalStyle.js

```diff
@@ -0,0 +1,20 @@
+import { createGlobalStyle } from 'styled-components';
+
+export default createGlobalStyle`
+  * {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
```

Comment:
Do you really wanna set margin and padding to 0 everywhere? I kind of see the idea behind it, yet I've never seen it before. :)